indico/modules/events/registration/controllers/display.py

```diff
@@ -490,6 +489,16 @@ def _process(self):
         return redirect(url)
 
 
+class RHTicketApplePass(RHTicketDownload):
+    """Download Apple Pass registration ticket."""
+
+    def _process(self):
+        if not (apple_pass := self.registration.generate_ticket_apple_pass()):
+            raise NotFound('Apple Pass tickets are not available')
```

Comment:
I would try to stick to the wording used by Apple in their official documentation (https://support.apple.com/en-us/111112). 

```suggestion
            raise NotFound('Ticket for Apple Wallet is not available')
```